scripts/run-encryption-tests-local.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
For both this and the other script - I think adding some comments / documentation explaining what they're for and how they accomplish their tasks would be useful.  MongoDB engineers are familiar with drivers-evergreen-tools, but people outside the company won't be.  Questions like: 

- what is the purpose of drivers-evergreen-tools?
- what it the output / how can you access information about the cluster?
- how can it be configured?

Basically anything that we think somebody who looks at this script might not understand.